src/main/java/kg/jarkyn/server/Response/ResponseContent.java

```diff
@@ -0,0 +1,20 @@
+package kg.jarkyn.server.Response;
+
+public class ResponseContent {
+    private byte[] byteContent;
+    private String contentType;
+
+    public ResponseContent(byte[] byteContent, String contentType) {
+
+        this.byteContent = byteContent;
```

Comment:
I think Christoph was referring to the name `byteContent` which does indicate the type in the name :)